gateway-spi/src/main/java/org/apache/knox/gateway/i18n/GatewaySpiMessages.java

```diff
@@ -93,4 +96,22 @@ public interface GatewaySpiMessages {
 
   @Message(level=MessageLevel.DEBUG, text="Ignoring cookie path scope filter for default topology")
   void ignoringCookiePathScopeForDefaultTopology();
+
+  @Message(level=MessageLevel.DEBUG, text="Loaded proxy groups ACLs: {0}")
+  void loadedProxyGroupsAcls(String acls);
+
+  @Message(level = MessageLevel.ERROR, text = "Unauthorized connection for super-user: {0} from IP remoteAddress {1}")
```

Comment:
Suggestion: "User impersonation failed for user {0}. Connections from remote address {1} are not authorized."